bats/fb-katello-content.bats

```diff
@@ -116,6 +116,54 @@ setup() {
     --content-view="${CONTENT_VIEW}" --to-lifecycle-environment="${LIFECYCLE_ENVIRONMENT}" --from-lifecycle-environment="Library"
 }
 
+@test "export content view version" {
```

Comment:
need to add skips if not pulp3

Reply:
looks like I need to add a `tIsPulp3`.  `tIsPulp2` checks for pulp2 with 

```
tPackageExists pulp-server
```

What package should I check for presence of pulp3?

Reply:
@parthaa thoughts?

Reply:
@ehelms pointed out a way to me. Lemme check

Reply:
https://github.com/theforeman/forklift/blob/master/bats/foreman_helper.bash#L31-L36 use that (```tSkipIfPulp3Only```)

Reply:
@parthaa wait, that would do the opposite of what I want.  I want to skip if pulp3 is _not_ available.

Reply:
Added `tSkipIfNoPulp3`.